apps/meteor/client/views/room/body/UploadProgress/UploadProgressIndicator.tsx

```diff
@@ -53,7 +53,7 @@ const UploadProgressIndicator = ({ id, name, percentage, error, onClose }: Uploa
 			className={customClass}
 		>
 			<Box withTruncatedText zIndex={2} borderRadius={4}>
-				[{percentage}%] {name}
+				[{percentage}%] {name} {error && ` - ${error}`}
```

Comment:
any chance of this to be an empty string? if so, lets force boolean
```suggestion
				[{percentage}%] {name} {Boolean(error) && ` - ${error}`}
```